state/convert.go

```diff
@@ -579,18 +579,50 @@ func verifyBlockHash(spork *config.Spork, hash []byte, height uint64, hdr *entit
 	}
 	resultHash := flow.MerkleRoot(resultIDs...)
 	resultHashV5 := flow.MerkleRoot(resultIDsV5...)
```

Comment:
please do a similar refactor for the execution results hash. once that's done, you no longer need to differentiate between `resultHash` and `resultHashV5` since it's handled during generation

Reply:
According to todo comment:
` todo: remove this double check after mainnet23 spork, and have just one call to convertExecutionResult`
I removed double `resultHash` calculation, cause I checked with new flow-go version `resultHash` and `resultHashV5` are similar.

Reply:
Looking at the rest of the code, I think we should keep that logic and extend the version branching done with other types.

Reply:
@UlyanaAndrukhiv I tracked down what happened. It seems like this logic was only needed on devnet to fix an unintentional change to the chunk structure. That change was reverted before the mainnet spork, so it's safe to remove from rosetta.